content/javascript/concepts/functions/functions.md

```diff
@@ -27,7 +27,7 @@ Function declarations are used to create named functions. These functions can be
 The example code provided contains a function named `add()` that takes in 2 values and prints the sum of those numbers:
 
 ```js
-function add(number1, number2) {
+function sum(number1, number2) {
```

Comment:
I actually prefer this renaming as well 👍 let's make sure to change the corresponding explanation on line 27 to match our newly named function.

Reply:
@Ethan-Master-Coding I've added a suggestion. Please take a look here.